apps/easypid/src/features/wallet/FunkeWalletScreen.tsx

```diff
@@ -47,8 +47,9 @@ export function FunkeWalletScreen() {
       </YStack>
 
       <FlexPage safeArea="y" fg={1} flex-1={false} bg="transparent">
-        <XStack pt="$2">
+        <XStack pt="$2" jc="space-between">
           <IconContainer bg="white" aria-label="Menu" icon={<HeroIcons.Menu />} onPress={pushToMenu} />
+          {isParadymWallet() && <InboxIcon />}
```

Comment:
This should use feature flag instead. If didcomm is supported we want to show the inbox (it will become easier if we have another variant of the wallet later)